build-system/scripts/request_spot

```diff
@@ -24,7 +24,7 @@ INSTANCE_TYPE_SUFFIX=${cpu_map[$CPUS]}
 
 # Check if INSTANCE_TYPE_SUFFIX is set, if not, the CPU count is not recognized.
 if [ -z "$INSTANCE_TYPE_SUFFIX" ]; then
-  echo "Unrecognized CPU count: $CPUS"
+  >&2 echo "Unrecognized CPU count: $CPUS"
```

Comment:
>&2 at end of line seems to read better to me but ymmv

Reply:
Having it at the start of the line makes it very clear I think. Ones brain can't fail to see it.